drivers/health_flexible_service/app/views/health_flexible_service/my_vprs/_patient_card.haml

```diff
@@ -1,17 +1,17 @@
-- url = client_health_flexible_service_vprs_path(patient.client)
+- url = client_health_flexible_service_vprs_path(client)
 
 .card.c-card-overflow-inherit.d-block
   .card-content.px-4.pt-4.pb-2
     %header.d-flex.flex-wrap.row
       .col-md-12.col-lg-3.pr-0
         %h1.flex-grow-1
-          = link_to_if patient.client.present?, patient.name, url
+          = link_to client.name, url
         .mb-4
           .c-detail-block
             .c-detail-block__title
               DOB
             .c-detail-block__value
-              = "#{patient.birthdate} (#{patient.age(on_date: Date.current)})"
+              = "#{client.DOB} (#{client.age})"
```

Comment:
same as above, I wonder if it makes sense to _try_ to use the patient data if we have it.